changelog.d/8648.bugfix

```diff
@@ -0,0 +1 @@
+Fix an issue that would cause ephemeral events to not be sent to appservices.
```

Comment:
I don't know if it makes sense to include that this was in rc1? I.e. it was never released.

Reply:
Release candidates have changelogs and rc1 was released. For example, [1.21.0rc2](https://github.com/matrix-org/synapse/releases/tag/v1.21.0rc2) includes an entry saying "Fix a regression in v1.21.0rc1 which broke thumbnails of remote media"